Service/LoggerManagement.php

```diff
@@ -84,7 +84,7 @@ public function addEvent(
         $log->setSeverity($severity);
         $log->setIdentifierLabel($identifierLabel);
         $log->setIdentifierValue($identifierValue);
-        $log->setTimestamp(strtotime(time()));
+        $log->setTimestamp(time());
```

Comment:
what about https://www.rakeshjesadiya.com/current-time-for-timestamp-and-datetime-type-in-magento-object-save-repository/

Reply:
@bartoszkubicki That's also working but for that, we need to modify the DB schema column.
I just want to prevent Database table column modification and added the current time programmatically.

Can We modified the current DB schema and fixed it?